Makefile

```
@@ -0,0 +1,18 @@
+.PHONY: all
+all: check
+
+.PHONY: check
+check:
+	cargo +nightly fmt --check
+	cargo audit
+	cargo clippy
```

I always do this, to apply it to the tests, too:

```suggestion
	cargo clippy --all-targets
```

---

👍 I had done the same in my open PR

---

Fixed in 0f82202f54030ce9a9769ea4d8e4c3ac1dbec4bd